test/url.jsx

```diff
@@ -0,0 +1,59 @@
+import { expect } from 'chai';
+import React from 'react';
+
+import { url } from '../';
+
+import callValidator from './_callValidator';
+
+describe('url', () => {
+  const validURL = 'https://www.airbnb.com';
+  const invalidURL = 'foobar';
```

Comment:
technically this is a valid relative URL; what about `/foo`?

If this propType is going to always require a fully qualified URL, i don’t think it’s useful - the vast majority of links on a site are internal, and internal links should always be relative-absolute (iow, start with a slash)